metashare/api/sf_run_flow.py

```diff
@@ -304,3 +304,26 @@ def create_org_and_run_flow(
         flow.run(org_config)
 
     return scratch_org_config
+
+
+def delete_scratch_org(scratch_org):
+    # Update environment with some keys CumulusCI uses
+    # TODO: Should this be in a context manager so as not to leak across
+    # runs?
+    os.environ.update(
+        {
+            # Salesforce connected app
+            "SFDX_CLIENT_ID": SF_CLIENT_ID,
+            "SFDX_HUB_KEY": SF_CLIENT_KEY,
+        }
+    )
+
+    devhub_username = scratch_org.owner.sf_username
+    org_id = scratch_org.config["org_id"]
+
+    devhub_api = get_devhub_api(devhub_username=devhub_username)
+    active_scratch_org_id = devhub_api.query(
+        f"SELECT Id FROM ActiveScratchOrg WHERE ScratchOrg='{org_id}'"
+    )["records"][0]["Id"]
```

Comment:
Need to handle the case where this returns no records because the scratch org was already deleted some other way.